src/device_bindings.cpp

```diff
@@ -124,9 +124,26 @@ void init_binding_device(pybind11::module& m){
             "Returns a vector defining how much the right camera is translated w.r.t left camera."
         )
 
+        .def(
+            "is_usb3",
+            &Device::is_usb3,
+            "Return true if connected over usb3 or else false."
+        )
+
+        .def(
+            "get_mx_id",
+            &Device::get_mx_id,
+            "Return the Myraid X serial number of the device."
+        )
 
-        
-        ;
+        .def("deinit",
```

Comment:
Pybind automatically calls objects destructor once the condition for destrucition are met. (Eg reference count goes to zero, ...). 
Calling destructor manually here will most likely call it twice in the end causing issues

Reply:
Yes. that was a mistake. added it while trying to overcome the error of deleting the device obj and realized it later that it won't work that way. Will be removing it in next push along with other changes.